test/servinge2e/deprecated_resources_test.go

```diff
@@ -52,3 +52,15 @@ func TestDomainMappingResource(t *testing.T) {
 		}
 	})
 }
+
+// SRVKS-1264 - cleanup of old internal TLS secrets
+func TestOldTLSSecret(t *testing.T) {
```

Comment:
Given that 1.11 (1.32) is the last time the secret was deployed I am wondering if the test is useful here. I think this matters for the 1.33 branch during upgrades.

Reply:
I was just hoping it's part of upgrade test, but haven't checked that.

Reply:
Are we planning to backport this?

Reply:
It's still worth to have in 1.33.z if we happen to release another z-stream patch.

Reply:
Ok let's add it.